src/download.c

```diff
@@ -301,6 +301,7 @@ int untar_full_download(void *data)
 	if (err) {
 		printf("ignoring tar extract failure for fullfile %s.tar (ret %d)\n",
 		       file->hash, err);
+		err = 0;
```

Comment:
I agree it's helpful to see error messages for untar failures, but the intention _is_ to retry downloads in the failure case, and retries will not occur if the error code is reset to 0.

So I would prefer keeping the existing `err` value, and removing the `printf` on the line above. @tmarcu Thoughts?